src/executables/csql.c

```diff
@@ -1449,6 +1449,9 @@ csql_do_session_cmd (char *line_read, CSQL_ARGUMENT * csql_arg)
 	{
 	  csql_arg->single_line_execution = false;
 	}
+
+      fprintf (csql_Output_fp, "SINGLELINE IS %s\n", (csql_arg->single_line_execution ? "ON" : "OFF"));
```

Comment:
Print only when it is csql interactive (check for the variable **csql_Is_interactive** is **true**).

Reply:
I understand. I will apply it.